src/profile/SecurityKeyCard.vue

```diff
@@ -13,6 +13,7 @@
             :keyId="webauthnKey.id"
             :mfaId="mfaId"
             :readOnly="!!isSummary"
+            @isWebauth="true"
```

Comment:
Was this supposed to be `isWebauthn` (with the `n`)?

Reply:
Good eye. Unfortunate the tooling didn't catch this.